src/app/schemas/generation_template.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+from pydantic import BaseModel
+from pydantic.schema import datetime
+
+
+class GenerationTemplateBase(BaseModel):
+    name: str
+    description: Optional[str]
+    content: str
+
+
```

Comment:
```suggestion
    class Config:
        schema_extra = {
            "example": {
                "name": "Niryo One (Python 3, ROS Melodic)",
                "description": "Python 3 templated based on ROS Melodic for the Niryo One",
                "content":
                    "{% for p in welding_points %}"
                    "{{p.x}}, {{p.y}}, {{p.z}} / {{p.roll}}, {{p.pitch}}, {{p.yaw}} / {{p.welding_order}}"
                    "{% endfor %}"
            }
        }


```
Can we add a schema example here?

Reply:
Addressed